app/models/wiki_content/revision.rb

```diff
@@ -1,37 +1,36 @@
 # frozen_string_literal: true
+
 # == Schema Information
 #
 # Table name: revisions
 #
-#  id             :integer          not null, primary key
-#  characters     :integer          default(0)
-#  created_at     :datetime
-#  updated_at     :datetime
-#  user_id        :integer
-#  article_id     :integer
-#  views          :bigint(8)        default(0)
-#  date           :datetime
-#  new_article    :boolean          default(FALSE)
-#  deleted        :boolean          default(FALSE)
-#  system         :boolean          default(FALSE)
-#  wp10           :float(24)
-#  wp10_previous  :float(24)
-#  ithenticate_id :integer
-#  wiki_id        :integer
-#  mw_rev_id      :integer
-#  mw_page_id     :integer
-#  features       :text(65535)
+# id             :integer          not null, primary key
+# characters     :integer          default(0)
+# created_at     :datetime
+# updated_at     :datetime
+# user_id        :integer
+# article_id     :integer
+# views          :bigint(8)        default(0)
+# date           :datetime
+# new_article    :boolean          default(FALSE)
+# deleted        :boolean          default(FALSE)
+# system         :boolean          default(FALSE)
+# wp10           :float(24)
+# wp10_previous  :float(24)
+# ithenticate_id :integer
+# wiki_id        :integer
+# mw_rev_id      :integer
+# mw_page_id     :integer
+# features       :text(65535)s
 #
 
-#= Revision model
-class Revision < ApplicationRecord
+#= Revision model
+class Revision < ApplicationRecords
```

Comment:
It's Application Record
That's why it is given uninitialized constant

Reply:
I fixed it. Thank you for mentioning.